### PR TITLE
Use common header for remaining web views (source code, disassembly, peek)

### PR DESCRIPTION
**Pull request by ghemawat**
Details:

* Made it easier to reuse css, javascript, etc. across views.
* Added two new templates: sourcelisting, plaintext.
* Use new templates for showing source code, disassembly, peek.
* Merged Functions menu into View menu.
* View change menu entries are now always enabled.
* Removed now unused "newWindow" argument to navigate.
* Removed redundancy in web handlers.
* Renamed /weblist URL to /source.
* Assembly and source code listings are now limited to 500 functions
  sorted by decreasing flat value (to prevent huge delays when
  selecting asm/source view across the entire profile).

### CODE REVIEW

**Comment by ghemawat** [comment]
After Travis CI delays, this looks ready for review.

@aalexand
@rauls5382

**Comment by aalexand** [comment]
The changes look good to me. I've added @josef-jelinek and @rauls5382 to the reviewer list to see if they have any comments. I'll merge the change tomorrow afternoon if there are no further comments.

File: internal/driver/webhtml.go [hunk 1]
```diff
@@ -16,21 +16,18 @@ package driver
...
 import "html/template"
 
-// webTemplate defines a collection of related templates:
-//    css
-//    header
-//    script
-//    graph
-//    top
-var webTemplate = template.Must(template.New("web").Parse(`
+// addTemplates add a set of template defintions to templates.
```
[the remark this answers is not in the document]
Comment (ghemawat) [reply internal/driver/webhtml.go]: Done.

[the remark this answers is not in the document]
Comment (ghemawat) [reply internal/driver/webhtml.go]: Done.

File: internal/driver/webui.go [hunk 2]
```diff
@@ -271,108 +297,85 @@ func dotToSvg(dot []byte) ([]byte, error) {
...
-	ui.output(w, req, "peek", "text/plain", vars)
 }
 
-// output generates a webpage that contains the output of the specified pprof cmd.
-func (ui *webInterface) output(w http.ResponseWriter, req *http.Request, cmd, ctype string, vars variables) {
-	focus := req.URL.Query().Get("f")
-	if focus == "" {
-		fmt.Fprintln(w, "no argument supplied for "+cmd)
-		return
+// source generates a web page containing disassembly.
```
[the remark this answers is not in the document]
Comment (ghemawat) [reply internal/driver/webui.go]: Done.

**Pull request merged**